### PR TITLE
Allow disabling/skipping of workflow tasks via config

### PR DESCRIPTION
**Pull request by larsgeorge-db**
Implements #283

### CODE REVIEW

File: src/databricks/labs/ucx/config.py [hunk 1]
```diff
@@ -124,6 +125,7 @@ def from_dict(cls, raw: dict) -> "MigrationConfig":
             warehouse_id=raw.get("warehouse_id", None),
             num_threads=raw.get("num_threads", 10),
             log_level=raw.get("log_level", "INFO"),
+            skip_tasks=raw.get("skip_tasks", []),
```
Comment (nfx) [inline src/databricks/labs/ucx/config.py:128]: I am not convinced we need it.

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/config.py (hunk 1 above)
Comment (FastLee) [reply src/databricks/labs/ucx/config.py:128]: I am. Most customers will start with Assessment only where the other tasks may come weeks and sometimes Months later.

Comment (nfx) [reply src/databricks/labs/ucx/config.py:128]: then we need to split the jobs and not skip tasks.

Comment (FastLee) [reply src/databricks/labs/ucx/config.py:128]: I would like them to be able to start with a quick assessment not messing up with the other tasks. Then redeploy when they are ready, making sure they have the latest code.

Comment (nfx) [reply src/databricks/labs/ucx/config.py:128]: we can split the jobs with enough justification.

Comment (FastLee) [reply src/databricks/labs/ucx/config.py:128]: Some customers may want to not deploy the group migration. Even look at it as a risk.
The assessment is a read only job and considered low risk.

**Comment by larsgeorge-db** [comment]
Closing this for now, since this requires either a larger discussion on how to address slow tasks or how configure the assessment step. For now, refreshing the dashboard manually is ok.

**Pull request closed without merging**